THEME.md

```diff
@@ -132,14 +141,30 @@ The following will inject the action link (or button) into the e-mail:
 It's a good idea to add this to the top of the template to specify a fallback color for the action button in case it wasn't provided by the user:
 
 ```html
-<% 
+<%
 // Make it possible to override action button color (specify fallback color if no color specified)
-if (locals.action && !locals.action.button.color) { 
+if (locals.action && !locals.action.button.color) {
     locals.action.button.color = 'blue';
 }
 %>
 ```
 
+## Dictionary Injection
+
+The following will inject a `<dl>` of key-value pairs into the e-mail:
+
+```html
+<!-- Dictionary -->
+<% if (locals.dictionary) { %>
+    <dl>
+    <% for (item in dictionary) { -%>
+        <dt><%- item %></dt>
```

Comment:
If you could please replace this line with the following so that the item name is both capitalized and suffixed with a colon, that'd be great:

```html
<dt><%- item.charAt(0).toUpperCase() + item.slice(1) %>:</dt>
```